kivy/uix/textinput.py

```diff
@@ -2460,32 +2705,27 @@ def _key_up(self, key, repeat=False):
             self._alt_r = False
 
     def keyboard_on_key_down(self, window, keycode, text, modifiers):
-        # Keycodes on OS X:
-        ctrl, cmd = 64, 1024
-        key, key_str = keycode
+        key, _ = keycode
         win = EventLoop.window
 
         # This allows *either* ctrl *or* cmd, but not both.
         modifiers = set(modifiers) - {'capslock', 'numlock'}
-        is_shortcut = (modifiers == {'ctrl'} or (
-            _is_osx and modifiers == {'meta'}))
-        is_interesting_key = key in (list(self.interesting_keys.keys()) + [27])
-
-        if not self.write_tab and super(TextInput,
-            self).keyboard_on_key_down(window, keycode, text, modifiers):
-            return True
+        is_shortcut = (
+            modifiers == {'ctrl'}
+            or _is_osx and modifiers == {'meta'}
```

Comment:
```suggestion
            'ctrl' in modifiers and 'meta' not in modifiers
            or _is_osx and 'ctrl' not in modifiers and 'meta' in modifiers
```
I think this is safer because numlock/capslock may not be the only modifiers active, but we can't know what they would be. But I don't feel very strongly about this.